proprietary_plugins/02_pro/general_enhancements.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "General Enhancements"
+description: "Further enahncements to "
```

Comment:
Wow I totally forgot to finish this sentence 😅 

Since this is not just for findings, maybe something like 
> Further enhancements to already great features

Reply:
Works for me!